DisplayTable.cs

```diff
@@ -0,0 +1,28 @@
+﻿using PhoneBookApp.Controllers;
+using PhoneBookApp.Data;
+using PhoneBookApp.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PhoneBookApp
+{
+    internal class DisplayTable
```

Comment:
Display table and format table should be different methods in the same class. Think of classes as groups of methods related to achieving a goal. In this case, the goal is to present data to the user.